.changelog/v0.8.0/summary.md

```diff
@@ -0,0 +1,11 @@
+*Apr 25, 2023*
+
+This release bumps the supported version of RocksDB, which requires cometbft-db
+RocksDB users to update their builds (and hence requires a "major" release, but
+does not introduce any other breaking changes). Special thanks to @yihuang for
+this update!
+
+While the minimum supported version of the Go compiler was bumped to 1.19, no
+1.19-specific code changes were introduced and this should, therefore, still be
+able to be compiled with earlier versions of Go. It is, however, recommended to
```

Comment:
Would it make sense to add a sentence here saying that 1.19 is the version used by the core team for all our testing? (CI, e2e, pre-release QA)

Reply:
Is that not covered in the README by stating the minimum Go version? https://github.com/cometbft/cometbft-db#minimum-go-version